### ISSUE TITLE
The preview PDF has a particularly low resolution

### ISSUE THREAD

**Issue by thunderbolt-fire** [freddyaboulton/gradio-pdf#10]
![image](https://github.com/user-attachments/assets/044d9274-5657-4251-a9d4-0dd8da7141ba)

I can't see the pdf clearly.

how to improve its resolution

**Comment by freddyaboulton** [comment]
Do you mind sharing that PDF @thunderbolt-fire ?

**Comment by thunderbolt-fire** [comment]
![image](https://github.com/user-attachments/assets/7f40c96c-f97a-4069-8db8-b0cb7536beee)
After zooming in, I found it even less clear. I try many pdfs but also like it.


```
import gradio as gr
from gradio_pdf import PDF

with gr.Blocks() as demo:
    pdf = PDF(label="Upload a PDF", interactive=True)
    name = gr.Textbox()
    pdf.upload(lambda f: f, pdf, name)

demo.launch()
```

**Comment by StudyingLover** [comment]
same problem, my pdf url is https://arxiv.org/pdf/2403.16971 .
I wang to build a paper chatbot but the resolution of  pdf is too low and I can't select words

**Comment by freddyaboulton** [comment]
Fixed in version 0.0.13
